apps/website/package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "eslint . --max-warnings 0",
```

Comment:
why do we need the `max-warnings` flag?

Reply:
We don't need this flag, is optional. 
This flag sets up a warning threshold, any amount of warnings above this threshold will make the lint command exit and fail. It just serves the purpose of having a buffer of the allowed amount of warnings in the project.

Things like our build step depend on this lint command.

Updated it to 5, if we get more than 5 warnings, our build will fail.